docs/forecasting.md

```diff
@@ -0,0 +1,26 @@
+# Forcasting
+
```

Comment:
Add a subsection called Naive forecasters. Currently we only have those. Inside that, we have these 2 algos.